tools/build-host-image.sh

```diff
@@ -86,11 +86,11 @@ function main
     if [ "x$BUILD_STANDARD" != "x" ]; then
         mkdir -p "$tmp_gz/root/root"
         echo "# Add opteadm, ddmadm to PATH" >> "$tmp_gz/root/root/.profile"
-        echo 'export PATH=$PATH:/opt/oxide/opte/bin:/opt/oxide/mg-ddm' >> "$tmp_gz/root/root/.profile"
+        echo "export PATH=$PATH:/opt/oxide/opte/bin:/opt/oxide/mg-ddm" >> "$tmp_gz/root/root/.profile"
```

Comment:
Missed this on my first pass, but this also is not meant to be expanded here.

Reply:
done in 90faa36